package.json

```diff
@@ -20,6 +22,7 @@
     "postcss": "^8.4.27",
     "tailwindcss": "^3.3.3",
     "typescript": "^5.1.6",
-    "vite": "^4.4.9"
+    "vite": "^4.4.9",
+    "vue-loader": "^17.4.2"
```

Comment:
```suggestion
```
Remove vue-loader, this was implemented while looking into styling for the markdown page.